### DISCUSSION TITLE
How do I use the Cake.AliaSQL dll?

### DISCUSSION THREAD

**Discussion by juleedev** [cake-build/cake#4296] [Extension Q&A] [answered]
Can someone help me answer the following question?  https://stackoverflow.com/questions/78104117/running-aliasql-on-mac
I basically have downloaded Cake.AliaSQL and compiled.  But I'm not familiar enough our tech stack to understand how to set things up so I can run AliaSQL commands against a local database, on my Mac.  Thanks in advance for your help.

**Comment by nils-a** [comment, marked as answer]
I answered on your SO question. 

If you are trying to use AliaSQL on a mac, Cake.AliaSQL can not help you, as it requires a running version of `AliaSQL.exe`.

The only options you have IMO is to either use [mono](https://www.mono-project.com/docs/getting-started/install/mac/) to run AliaSQL.exe on mac (not sure if that works, though) or port AliaSQL to a modern version of .NET.
